src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -465,7 +473,7 @@ object AppDefinition extends GeneralPurposeCombinators {
   val DefaultNetworks = Seq[Network](HostNetwork)
 
   def fromProto(proto: Protos.ServiceDefinition): AppDefinition =
-    AppDefinition(id = DefaultId).mergeFromProto(proto)
+    AppDefinition(id = DefaultId, role = null).mergeFromProto(proto)
```

Comment:
Should we comment why `null` is fine here?

Reply:
Good idea.